docker/webpackager/README.md

```diff
@@ -0,0 +1,307 @@
+# Web Packager Server
+
+Web Packager HTTP Server is an HTTP server built on top of Web Packager.
```

Comment:
Is this the text people will see in the marketplace? If so, then we should probably revise it to better suit that context (in a future PR) -- start a doc? Otherwise, ignore this comment.

Reply:
This is not for the marketplace, so ignoring the comment.